### PR TITLE
add import subcommand

### PR DESCRIPTION
**Pull request by vvakame**
hi, I want to use `terraform import fastly_service_v1.xxx xxxx`.
because I want to execute the `plan` after getting the same state as the real fastly service.

### CODE REVIEW

**Comment by vvakame** [comment]
or use backend?

**Comment by cauealvesbraz** [comment]
Hello @vvakame, thanks for this submission.

I have the same issue when trying to use `terraform plan ...` on step. 

Did you solve this issue without a backend implementation?

**Comment by cauealvesbraz** [comment]
Hey @sudomateo, how are you?

First for all, congrats for this project :clap: 

Did you have any update about this Pull Request?

Adding ` terraform import` as subcommand is very useful 👍

**Comment by vvakame** [comment]
I'm using GCS backend finally...

**Comment by cauealvesbraz** [comment]
I'm using the AWS as provider and I didn't want to create a custom step for use `terraform import`... 

Any concern for this Pull Request @sudomateo?

File: src/terraform_import.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+  # Exit code of !0 indicates failure.
+  if [ ${importExitCode} -ne 0 ]; then
+    echo "import: error: failed to import Terraform configuration in ${tfWorkingDir}"
+    echo "${importOutput}"
+    echo
+  fi
+
+  # Comment on the pull request if necessary.
+  if [ "$GITHUB_EVENT_NAME" == "pull_request" ] && [ "${tfComment}" == "1" ] && [ "${planCommentStatus}" == "Failed" ]; then
```
Comment (sudomateo) [inline src/terraform_import.sh]:
```suggestion
  if [ "$GITHUB_EVENT_NAME" == "pull_request" ] && [ "${tfComment}" == "1" ] && [ "${importCommentStatus}" == "Failed" ]; then
```

Comment (cauealvesbraz) [reply src/terraform_import.sh]: Hey @vvakame, how are you? Could you do the proposed changes?

**Comment by cauealvesbraz** [comment]
@vvakame Will you be able to make the changes?

File: src/terraform_import.sh (hunk 1 above)
Comment (vvakame) [reply src/terraform_import.sh]: @sudomateo thanks! add your suggestion. thanks!

**Comment by cauealvesbraz** [comment]
@sudomateo Could you review again?

**Comment by cauealvesbraz** [comment]
Ping @sudomateo

**Comment by sudomateo** [comment]
@cauealvesbraz Appreciate the pings for visibility. I'll review this when I'm back in the office on Monday. Don't worry it's still on my radar.

**Pull request merged**